README.md

```diff
@@ -1,2 +1,311 @@
+  --log-format {json,pretty}
+                        Force the format of the logs. By default, if the
+                        command is from a terminal, print colorful logs.
+                        Otherwise print json.
+  --log-file LOG_FILE   Writes logs to log file if specified, default: None
+  --quiet               if true, does not print logs to stderr, default: False
+
+commands:
+  {collect,forward,run}
+    collect             Collects the logs from different processes and sends
+                        to nsq
+    forward             Collects all the messages from nsq and pushes to
+                        MongoDB
+```
+
+- For `logagg collect`
+```bash
+logagg collect --help
+```
+- You should see something like
+```bash
+usage: logagg collect [-h] [--nsqd-http-address NSQD_HTTP_ADDRESS]
+                      file [file ...] nsqtopic
+
+positional arguments:
+  file                  Provide absolute path of logfile including module name
+                        and function name, eg: /var/log/nginx/access.log:logag
+                        g.collect.handlers.nginx_access
+  nsqtopic              Topic name to publish messages. Ex: logs_and_metrics
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --nsqd-http-address NSQD_HTTP_ADDRESS
+                        nsqd http address where we send the messages
+```
+
+- For `logagg forward`,
+```bash
+logagg forward --help
+```
+- You should see something like
+```bash
+usage: logagg forward [-h] [--nsqtopic NSQTOPIC] [--nsqchannel NSQCHANNEL]
+                      [--nsqd-tcp-address NSQD_TCP_ADDRESS]
+                      [--mongodb-server-url MONGODB_SERVER_URL]
+                      [--mongodb-port MONGODB_PORT]
+                      [--mongodb-user-name MONGODB_USER_NAME]
+                      [--mongodb-password MONGODB_PASSWORD]
+                      [--mongodb-database MONGODB_DATABASE]
+                      [--mongodb-collection MONGODB_COLLECTION]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --nsqtopic NSQTOPIC   NSQ topic name to read messages from. Ex:
+                        logs_and_metrics
+  --nsqchannel NSQCHANNEL
+                        channel of nsqd
+  --nsqd-tcp-address NSQD_TCP_ADDRESS
+                        nsqd tcp address where we get the messages
+  --mongodb-server-url MONGODB_SERVER_URL
+                        DNS of the server where mongo is running
+  --mongodb-port MONGODB_PORT
+                        port where mongo is running
+  --mongodb-user-name MONGODB_USER_NAME
+                        username of mongodb
+  --mongodb-password MONGODB_PASSWORD
+                        password to authenticate mongodb
+  --mongodb-database MONGODB_DATABASE
+                        database to store logs
+  --mongodb-collection MONGODB_COLLECTION
+                        collection to store logs
+
+```
+
+### Run the `collector` 
```

Comment:
@praneeth36 @AnushaKonduru We should have this section title as `How to run log collector?`